tasks/insights-client.yml

```diff
@@ -10,6 +10,13 @@
   when:
     - rhc_state | d("present") != "reconnect"
 
+- name: Configure tags
+  copy:
+    dest: /etc/insights-client/tags.yml
+    content: "{{ rhc_insights.tags | to_nice_yaml }}"
+    mode: '0644'
+  when: rhc_insights.tags is defined
```

Comment:
i think that, since `tags` was added in `defaults/main.yml`, this check will be always true

Reply:
> i think that, since `tags` was added in `defaults/main.yml`, this check will be always true

It might not be defined - consider if someone uses `rhc_insights: {}`
this should work `rhc_insights.tags | d({}) | length > 0` - an empty `dict` should have a length of 0, otherwise, the length should be the number of keys